holidays/countries/macau.py

```diff
@@ -0,0 +1,461 @@
+        "Concelho das Ilhas": "I",
+        "海島市": "I",
+        "海岛市": "I",
+        "Concelho de Macau": "M",
+        "澳門市": "M",
+        "澳门市": "M",
+    }
+    supported_categories = (GOVERNMENT, MANDATORY, PUBLIC)
+    supported_languages = ("en_MO", "en_US", "pt_MO", "th", "zh_CN", "zh_MO")
+    # Decreto-Lei n.º 4/82/M.
+    start_year = 1982
+    weekend = {SAT, SUN}
+
+    def __init__(self, *args, **kwargs):
+        ChineseCalendarHolidays.__init__(self)
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, MacauStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_observed(self, dts: set[date], multiple: bool = False) -> None:
+        observed_label = (
+            # Compensatory rest day for %s.
+            self.tr("%s的補假")
+            if self._year >= 2020
+            # The first working day after %s.
+            else self.tr("%s後首個工作日")
+        )
+        for dt in sorted(dts):
+            if not self._is_observed(dt):
+                continue
+            for name in self.get_list(dt):
+                self._add_observed(dt, observed_label % name, show_observed_label=False)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("元旦"))
+
+        # Lunar New Year's Day.
+        self._add_chinese_new_years_day(tr("農曆正月初一"))
+
+        # The second day of Lunar New Year.
+        self._add_chinese_new_years_day_two(tr("農曆正月初二"))
+
+        # The third day of Lunar New Year.
+        self._add_chinese_new_years_day_three(tr("農曆正月初三"))
+
+        # Ching Ming Festival.
+        self._add_qingming_festival(tr("清明節"))
+
+        # Regulamento Administrativo n.º 4/1999 - Name changed in Chinese for Good Friday.
+        name = (
+            # Good Friday.
+            tr("耶穌受難日")
+            if self._year >= 2000
+            # Good Friday.
+            else tr("聖周星期五")
+        )
+        self._add_good_friday(name)
+
+        # Regulamento Administrativo n.º 4/1999 - Name changed to The Day before Easter.
+        name = (
+            # The Day before Easter.
+            tr("復活節前日")
+            if self._year >= 2000
+            # Holy Saturday.
+            else tr("聖周星期六")
+        )
+        self._add_holy_saturday(name)
+
+        # Labor Day.
+        self._add_labor_day(tr("勞動節"))
+
+        # Tung Ng Festival (Dragon Boat Festival).
+        self._add_dragon_boat_festival(tr("端午節"))
+
+        # Chung Yeung Festival (Festival of Ancestors).
+        self._add_double_ninth_festival(tr("重陽節"))
+
+        # The Day following Chong Chao (Mid-Autumn) Festival.
+        self._add_mid_autumn_festival_day_two(tr("中秋節翌日"))
+
+        # National Day of the People's Republic of China.
+        self._add_holiday_oct_1(tr("中華人民共和國國慶日"))
+
+        # All Soul's Day.
+        self._add_all_souls_day(tr("追思節"))
```

Comment:
I think that the special English names of these holidays should be left only in en_MO (to match official sources), and for en_US, simpler and more traditional ones should be used (perhaps with a little bit of a change, if necessary).

Reply:
I've standardize `en_US` ones to China holidays ones for now 🙏